tests/Serval.SpecFlowTests/Utilities.cs

```diff
@@ -0,0 +1,49 @@
+namespace Serval.SpecFlowTests;
+
+public class Utilities
+{
+    static async public Task<string> GetAccessTokenFromEnvironment(HttpClient httpClient)
+    {
+        var access_token = "";
+        var client_id = Environment.GetEnvironmentVariable("MACHINE_CLIENT_ID");
```

Comment:
All of the references to Machine should be changed to Serval.

Reply:
Done.